security/experimental_authenticators.rst

```diff
@@ -0,0 +1,432 @@
+Using the new Authenticator-based System
+========================================
+
+.. versionadded:: 5.1
+
+    The authenticator-based system was introduced as
+    :doc:`experimental feature </contributing/code/experimental>` in
+    Symfony 5.1.
+
+In Symfony 5.1, a new authentication system was introduced. This system
+changes the internals of the Security system, to make it more extensible
+and easier to understand.
```

Comment:
What about: more understandable
As imo, the code for an external pov can be not this easy :)